app/src/main/res/values-en/strings.xml

```diff
@@ -173,7 +173,7 @@ First off, understand that with this app, you make actual and direct contributio
 Anything you contribute in this app is directly added to the map, there is no third party in-between the app and OSM infrastructure.
 &lt;/p&gt;
 &lt;p&gt;
-Editing the OpenStreetMap anonymously is not possible, so any changes you make are made with your OSM user account. This means, that the date, content and implicitly the location of your changes are visible publicly on the openstreetmap website and are attributable to your account.
+Editing the OpenStreetMap anonymously is not possible, so any changes you make are made with your OSM user account. This means, that the date, content and the location of your changes are visible publicly on the openstreetmap website and are attributable to your account.
```

Comment:
@matkoniecz alternative:
Editing the OpenStreetMap anonymously is not possible, so any changes you make are made with your OSM user account. This means, that the date, content and the location of your changes are visible publicly on the openstreetmap website and are attributable to your account. It means that also with StreetComplete on-site survey and OpenStreetMap improvement reveals your location.

Reply:
I quite like @matkoniecz suggestion with a few minor tweaks:
```suggestion
Editing the OpenStreetMap anonymously is not possible, so any changes you make are made with your OSM user account. This means, that the date, content and the location of your changes are visible publicly on the OpenStreetMap website and are attributable to your account. It means that StreetComplete on-site survey and OpenStreetMap improvements reveal where you have been.```

Reply:
``` is likely not intended

Reply:
Indeed, fixed! That's an odd bug if you don't have the line break!